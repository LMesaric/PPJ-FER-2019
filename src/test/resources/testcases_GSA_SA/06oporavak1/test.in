a 1 a
b 3 b
b 4 b
a 5 a
b 7 b
