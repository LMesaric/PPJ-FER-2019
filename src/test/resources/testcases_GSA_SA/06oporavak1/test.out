<S>
 <A>
  a 1 a
 <A>
  a 5 a
 <S>
  <B>
   b 7 b
